operator/v1/types_csi_cluster_driver.go

```diff
@@ -281,6 +281,27 @@ type VSphereCSIDriverConfigSpec struct {
 	// will be rejected.
 	// +optional
 	TopologyCategories []string `json:"topologyCategories,omitempty"`
+
+	// globalMaxSnapshotsPerBlockVolume is a global configuration parameter that applies to volumes on all kinds of
+	// datastores. By default, it is set to three.
+	// +kubebuilder:validation:Optional
+	// +openshift:enable:FeatureSets=TechPreviewNoUpgrade
+	// +optional
+	GlobalMaxSnapshotsPerBlockVolume *uint32 `json:"globalMaxSnapshotsPerBlockVolume,omitempty"`
+
+	// granularMaxSnapshotsPerBlockVolumeInVSAN is a granular configuration parameter on vSAN datastore only. It
+	// overrides the global constraint if set, while it falls back to the global constraint if unset.
+	// +kubebuilder:validation:Optional
```

Comment:
IMO it would be better to use `overrides globalMaxSnapshotsPerBlockVolume` instead of `the global constraint`.